pkg/bugzilla/client.go

```diff
@@ -0,0 +1,80 @@
+// Package bugzilla implements a basic bugzilla jsonrpc api wrapper with a single query
+package bugzilla
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+// Bugzilla object with information to access the bugzilla api
```

Comment:
doc comments should be of the form

```go
// Noun verbs...
type Noun struct {}
```

so this would say `Bugzilla knows how to access the bugzilla API` or `Bugzilla is a client for the Bugzilla API` or `Bugzilla runs queries on the Bugzilla API`.  Notice that the object is named a bit redundantly.  You can just call it `Client`, because people will reference the type as `bugzilla.Client`.